src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -215,6 +216,106 @@ public static long Flt2LngOvf(float val)
             return ThrowIntOvf();
         }
 
+        [DllImport("libRuntime", EntryPoint="RhpIDiv")]
```

Comment:
These should be all `[RuntimeImport(RuntimeLibrary, ...)]`

Reply:
We have all RuntimeImports defined in RuntimeImport.cs. Maybe it is worth following the convention.